packages/expo-dev-launcher/android/src/main/java/expo/modules/devlauncher/helpers/DevLauncherJavaScriptExecutorExtension.kt

```diff
@@ -0,0 +1,70 @@
+package expo.modules.devlauncher.helpers
```

Comment:
You don't need that file - you can use a helper that was declared in the `expo-dev-menu` package directly.